### PR TITLE
Fix GFF parser

### PR DESCRIPTION
**Pull request by toothsy**
this commit helps to fix the error caused in issue #153, manually ran `go test ./...` everything worked fine

### CODE REVIEW

File: io/gff/gff.go [hunk 1]
```diff
@@ -206,6 +209,27 @@ func Parse(file io.Reader) (Gff, error) {
...
+// throws error if not found
+func extractInfoFromField(metaString []string, fieldName string) ([]string, int, error) {
+	index := 0
+	endOfMetaInfo := 0
+	for i, line := range metaString {
+		if strings.Contains(line, "#") {
+			if strings.Contains(line, fieldName) {
+				index = i
+			}
+		} else {
```
Comment (matiasinsaurralde) [inline io/gff/gff.go:222]: You could add `continue`, remove `else` and unindent that block:
```go
	for i, line := range metaString {
		if strings.Contains(line, "#") {
			if strings.Contains(line, fieldName) {
				index = i
			}
			continue
		}
		endOfMetaInfo = i
		break
	}
```

File: data/Arthrospira_platensis.gff
Comment (TimothyStiles) [inline data/Arthrospira_platensis.gff:1]: this is a pretty large file to add. Is there other test data already in Poly that can be used?

Comment (TimothyStiles) [reply data/Arthrospira_platensis.gff:1]: > this is a pretty large file to add. Is there other test data already in Poly that can be used?

ah wait this is @Koeng101's doing. What's the section that was giving you trouble @Koeng101? could we make a smaller test file for this?

Comment (toothsy) [reply data/Arthrospira_platensis.gff:1]: will add more accurate meta information to existing gff file, that oughta cover this particular scenario

Comment (toothsy) [reply data/Arthrospira_platensis.gff:1]: [sources.txt](https://github.com/TimothyStiles/poly/files/11059235/sources.txt)
found info from these sources, updated ecoli-mg1655.gff accordingly,

**Comment by TimothyStiles** [comment]
This PR was excellent @toothsy! Merged!!! 🤘

**Comment by toothsy** [comment]
Thank you @TimothyStiles, happy to contribute😁😁

**Pull request merged**